src/view/edit-event.js

```diff
@@ -1,10 +1,126 @@
 import { getTotalPrice } from '../utils/useRender';
-import { destinations } from '../mock/destinations';
 import dayjs from 'dayjs';
 
-export const createEditEventTemplate = (pointEvent, descriptionEvent) => {
-  const currentDate = dayjs().format('DD/MM/YY HH:mm');
+const createTypesTemplate = (currentType) => {
+  const typesTemplate = [];
+
+  for (const pointType of POINT_TYPES) {
+    const typeToRender = (
+      `<div class="event__type-item">
+        <input
+          id="event-type-${pointType.toLowerCase()}-1"
+          class="event__type-input visually-hidden"
+          type="radio" name="event-type"
+          value="${pointType.toLowerCase()}"
+          ${pointType === currentType ? 'checked' : ''}
+        >
+        <label
+          class="event__type-label event__type-label--${pointType.toLowerCase()}"
+          for="event-type-${pointType.toLowerCase()}-1"
+        >
+          ${pointType}
+        </label>
+      </div>`
+    );
+
+    typesTemplate.push(typeToRender);
+  }
+
+  return typesTemplate.join('');
+};
+
+const createDestinationOptionsTemplate = (allDestinations) => {
+  const destinationOptionsTemplate = [];
+
+  for (const destinationOption of allDestinations) {
+    const destinationOptionToRender = `<option value="${destinationOption}"></option>`;
+    destinationOptionsTemplate.push(destinationOptionToRender);
+  }
+
+  return destinationOptionsTemplate.join('');
+};
+
+const createOfferOptionsTemplate = (allOffers, offersType) => {
+  if (allOffers.length < 1) {
+    return '';
+  }
+
+  const offersTemplate = [];
+
+  for (const offerOption of allOffers) {
+    const offerForAttribute = `${offersType.replaceAll(' ', '-')}-${offerOption.id}`;
+
+    const offerToRender = (
+      `<div class="event__offer-selector">
+        <input
+          class="event__offer-checkbox visually-hidden"
+          id="event-offer-${offerForAttribute}"
+          type="checkbox"
+          name="event-offer-${offerForAttribute}"
+          ${offerOption.isAdded ? 'checked' : ''}
+        >
+        <label
+          class="event__offer-label"
+          for="event-offer-${offerForAttribute}"
+        >
+          <span class="event__offer-title">${offerOption.title}</span>
+          &plus;&euro;&nbsp;
+          <span class="event__offer-price">${offerOption.price}</span>
+        </label>
+      </div>`
+    );
+
+    offersTemplate.push(offerToRender);
+  }
+
+  return (
+    `<section class="event__section event__section--offers">
+      <h3 class="event__section-title event__section-title--offers">Offers</h3>
+      <div class="event__available-offers">
+        ${offersTemplate.join('')}
+      </div>
+    </section>`
+  );
+};
+
+const createPhotosTemplate = (allPhotos) => {
+  if (allPhotos.length < 1) {
+    return '';
+  }
+
+  const photosTemplate = [];
+
+  for (const photo of allPhotos) {
+    const photoToRender = `<img class="event__photo" src="${photo}" alt="Event photo">`;
```

Comment:
```suggestion
    const photoToRender = (
      `<img 
        class="event__photo" 
        src="${photo}" 
        alt="Event photo"
      />
    )`;
```
Ну и хоть для одинарных тегов не обязательно, но я бы ставил закрывающую `/`
Также удобней будет?)